perspective.el

```diff
@@ -663,11 +663,11 @@ buffer called \"*scratch* (NAME)\"."
 Returns BUFFERS with all non-living buffers removed.
 
 See also `other-buffer'."
-  (cl-loop for buf in (reverse buffers)
-           when (buffer-live-p buf)
-           collect buf into living-buffers
+  (cl-loop for buf in (reverse (buffer-list))
+           when (member buf buffers)
```

Comment:
Let's keep the check for `buffer-live-p` here (`(and (buffer-live-p buf) (member buf buffers))` or something like it). It might be overkill, but Perspective had a lot of "dead buffer" bugs over the years and I'd rather be more cautious.

Reply:
Thanks. Updated.